templates/event/event_show.html.twig

```diff
@@ -1,118 +1,128 @@
 
-        {% if (event.contactSite or event.contactFB or event.contactVK or event.contactTelegram or event.contactOther) %}
-            <p class="p-1"> <strong> Ссылки: </strong>
-                {% if event.contactSite %}
-                    <a href="http://{{ event.contactSite }}" class="text-dark mx-3"> сайт </a>
+                {% if event.genre %}
+                    <p class="pl-1"> <strong> Жанр:</strong> {{ event.genre.name }} </p>
                 {% endif %}
-                {% if event.contactFB %}
-                    <a href="{{ event.contactFB }}" class="text-dark"> <span class="fa fa-facebook-square mx-3" ></span></a>
+
+                {% if event.subgenres %}
+                    <p class="pl-1"> <strong> Поджанры: </strong>
+                        {% for subgenre in event.subgenres%}
+                            {% if loop.last %}
+                                {{ subgenre.name }}
+                            {% else %}
+                                {{ subgenre.name }},
+                            {% endif %}
+                        {% endfor %}
+                    </p>
                 {% endif %}
-                {% if event.contactVK %}
-                    <a href="{{ event.contactVK }}" class="text-dark"> <span class="fa fa-vk mx-3" ></span></a>
+
+                {% if event.organizers %}
+                    <p class="pl-1"> <strong> Организаторы:</strong> {{ event.organizers }} </p>
                 {% endif %}
-                {% if event.contactTelegram %}
-                    <a href="{{ event.contactTelegram }}" class="text-dark"> <span class="fa fa-telegram mx-3" ></span></a>
+
+                {% if event.settlement %}
+                    <p class="pl-1"> <strong> Тип поселения:</strong> {{ event.settlement.name }} </p>
+                {% endif %}
+
+                {% if not event.weapons is empty  %}
+                    <p class="pl-1"> <strong> Материал оружия: </strong>
+                        {% for weapon in event.weapons %}
+                            {% if loop.last %}
+                                {{ weapon.name }}
+                            {% else %}
+                                {{ weapon.name }},
+                            {% endif %}
+                        {% endfor %}
+                    </p>
                 {% endif %}
-                {% if event.contactOther %}
-                    <p>{{ event.contactOther }}</p>
+
+                {% if (event.priceMin or event.priceMax) %}
+                    <p class="p-1">
+                        <strong>Взнос: </strong>
+                        {% if event.priceMin %}
+                            {{ event.priceMin }}
+                            {% if event.priceMax and event.priceMax != event.priceMin %}
+                                - {{ event.priceMax }}
+                            {% endif %}
+                        {% else %}
+                            {{ event.priceMax }}
+                        {% endif %}
```

Comment:
@daria-sieroshtan 
This is a strange way to do it.
Better make `priceMin` mandatory and `priceMax` mandatory

Reply:
There can be no price data at all. It is often unknown at the time of event announcement.

Reply:
@daria-sieroshtan 
There is a way to make it more elegantly, but let it be as is